src/parsing/parsers/generic/jison/explain/explain.test.json

```diff
@@ -0,0 +1,95 @@
+[
+  {
+    "namePrefix": "should not report errors on EXPLAIN SELECT statement",
+    "beforeCursor": "EXPLAIN SELECT * FROM test_table; ",
+    "afterCursor": "",
+    "noErrors": true,
+    "containsKeywords": [
+      "SELECT"
+    ],
+    "expectedResult": {
+      "lowerCase": false
+    }
+  },
+  {
+    "namePrefix": "should not report errors on EXPLAIN DELETE statement",
+    "beforeCursor": "EXPLAIN DELETE FROM test_table; ",
+    "afterCursor": "",
+    "noErrors": true,
+    "containsKeywords": [
+      "SELECT"
+    ],
+    "expectedResult": {
+      "lowerCase": false
+    }
+  },
+  {
+    "namePrefix": "should not report errors on EXPLAIN CREATE statement",
+    "beforeCursor": "EXPLAIN CREATE TABLE Persons (id int); ",
+    "afterCursor": "",
+    "noErrors": true,
+    "containsKeywords": [
+      "SELECT"
+    ],
+    "expectedResult": {
+      "lowerCase": false
+    }
+  },
+  {
+    "namePrefix": "should report error on double EXPLAIN statement",
+    "beforeCursor": "EXPLAIN EXPLAIN SELECT * FROM test_table; ",
+    "afterCursor": "",
+    "noErrors": false,
+    "expectedResult": {
+      "lowerCase": false,
+      "suggestKeywords": [
+        "ALTER", "CREATE", "DELETE", "DESCRIBE", "DROP", "EXPLAIN", "GRANT", "INSERT", "REVOKE", "SELECT", "SET", "SHOW", "TRUNCATE", "UPDATE", "USE", "WITH"
+      ]
+    },
```

Comment:
We should remove this, because in our test we don't care about the result, only about the error